doc/source/ray-observability/ray-logging.rst

```diff
@@ -152,12 +152,12 @@ A new Ray instance creates a new session ID to the temp directory. The latest se
 
 Here's a Ray log directory structure. Note that ``.out`` is logs from stdout/stderr and ``.err`` is logs from stderr. The backward compatibility of log directories is not maintained.
 
-- ``dashboard.log``: A log file of a Ray dashboard.
+- ``dashboard.[log|out|err]``: A log file of a Ray dashboard. ``log.`` file contains logs generated from the dashboard's logger. ``.out`` and ``.err`` are stdout and stderr printed from the dashboard. They are usually empty except when the dashboard crashes unexpectedly.
```

Comment:
Will there be `out` files for dashboard and log_monitor? Thought we are redirecting stdout and stderr to a single `.err` file.

Reply:
ah good catch. Fixign it now..